packages/communication-react/src/mergedHooks.ts

```diff
@@ -109,10 +135,16 @@ export type ComponentProps<Component extends (props: any) => JSX.Element> = Chat
  *
  * @public
  */
-export const usePropsFor = <Component extends (props: any) => JSX.Element>(
+export function usePropsFor<Component extends (props: any) => JSX.Element>(
```

Comment:
For consistency - Instead of changing the const arrow function to normal function, can we type the const arrow function better like what is in this thread:
https://stackoverflow.com/questions/39187614/typescript-overload-arrow-functions

type UsePropsFor = {
    <Component extends (props: any) => JSX.Element>component: Component,
       type?: 'calling' | 'chat'
     ): ComponentProps<Component>; // main entry point

   /* @conditional-compile-remove(PSTN-calls) */
    <T>component: typeof HoldButton,
       type?: 'calling'
     ): HoldButtonSelector & Common<CallingHandlers, HoldButtonProps>; // fcuntion override
}

export const usePropsFor: UsePropsFor = () => {
} // implementation

Reply:
Consistency in free functions in our public API, do you mean?

I have a slight preference to `function` over arrow functions (because tsdoc deals with them better), but consistency would win in this argument.